packages/twenty-front/src/modules/types/SettingsPath.ts

```diff
@@ -18,6 +18,7 @@ export enum SettingsPath {
   NewObject = 'objects/new',
   WorkspaceMembersPage = 'workspace-members',
   Workspace = 'workspace',
+  CRMMigration = 'crm-migration',
```

Comment:
Not needed for merge but in future iterations I think we should rename it into something more generic like "import data" and also advert the CSV import in that tab because many people don't find it and it would be a good entry point to have it in settings